app/controllers/items_controller.rb

```diff
@@ -0,0 +1,62 @@
+class ItemsController < ApplicationController
+  before_action :set_item, only: %i[show edit update destroy]
+  before_action :authenticate_user!
+
+  def index
+    @items = Item.all
+  end
+
+  def show; end
+
+  def new
+    @item = Item.new
+    @group = Group.find(params[:group_id])
+  end
+
+  def edit; end
+
+  def create
+    @item = Item.new(item_params)
+    @item.user = current_user
+    @group = Group.find(params[:group_id])
+    respond_to do |format|
+      if @item.save
+        format.html { redirect_to group_path(@group), notice: 'Item was successfully created.' }
+        format.json { render :show, status: :created, location: @item }
+      else
+        format.html { render :new }
+        format.json { render json: @item.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  def update
```

Comment:
- [ ] [optional] it would be better to style the transaction page as well as in the given design